### ISSUE TITLE
Elements are not accessable via Keyboard. NVDA Screen reader does not read it properly

### ISSUE THREAD

**Issue by repl-prajacta-modani** [bgrins/spectrum#572]
It will be nice if this is accessable via keyboard ie Web Content Accessibility Guidelines (WCAG) 2.1 compliant. (https://www.w3.org/TR/WCAG21/ )
